moped-editor/src/views/projects/projectView/ProjectComponents/utils/map.css

```diff
@@ -0,0 +1,20 @@
+.mapbox-gl-draw_line.active {
```

Comment:
maybe add a comment here with a link to where you sourced these SVGs from?

Reply:
i sourced them from the dev tools inspector and then edited the svg....i bet i could find a better source for them in the mapbox repo?